### DISCUSSION TITLE
Maui and Multiple Windows?

### DISCUSSION THREAD

**Discussion by pjmagee** [dotnet/maui#1773] [Q&A]
I am using Maui blazor for the main app. 
I would like the ability to launch a 2nd Window which can host a Browser Control for Twitch Authentication and then once I retrieve the Token i need I can store this and close this second Window.

It doesnt look possible to launch multiple windows in Maui?

**Comment by vhugogarcia** [comment]
I think you can do it yeah!! By using MAUI.Essentials you should be able to do that 👍🏻

**Reply by mcguu** [reply]
I think he can do it too . yeah!!👍🏻
